services/httpd/handler.go

```diff
@@ -81,14 +81,16 @@ type Handler struct {
 		WritePoints(database, retentionPolicy string, consistencyLevel models.ConsistencyLevel, points []models.Point) error
 	}
 
-	// Normal wlog logger
-	logger *log.Logger
+	logService logging.Interface
+
+	// Normal logger
+	logger zap.Logger
 	// Detailed logging of write path
 	// Uses normal logger
 	writeTrace bool
 
 	// Common log format logger.
-	// This logger does not use log levels with wlog.
+	// This logger does not use log levels.
 	// Its simply a binary on off from the config.
 	clfLogger *log.Logger
```

Comment:
Yes, for now anyways. But I would like to more off of it at some point so that all logs have the same format.